components/usage-api/usage/v1/billing.proto

```diff
@@ -19,6 +19,9 @@ service BillingService {
   // FinalizeInvoice marks all sessions occurring in the given Stripe invoice as
   // having been invoiced.
   rpc FinalizeInvoice(FinalizeInvoiceRequest) returns (FinalizeInvoiceResponse) {};
+
+  // MarkWorkspaceInstanceSessionAsBilled marks an instance as billed with a billing system
+  rpc MarkWorkspaceInstanceSessionAsBilled(MarkWorkspaceInstanceSessionAsBilledRequest) returns (MarkWorkspaceInstanceSessionAsBilledResponse) {};
```

Comment:
IMO, you could go with a simpler CRUD like approach make it:
```
UpdateBilledSession(...)
```
or if you'd like to make it more explicit, then I'd go for 
```
SetWorkspaceSessionsBilled()
```

I personally favour the simpler CRUD like APIs in the form of `Update<ResourceName>` where the payload specifies what you're updating.

Reply:
The general principle in API design is that the API needs to be useful, but we should also limit the APIs surface. Fewer RPCs, which are more powerful (like Update<Resource>) make it easier to understand what you should use, and how to use it correctly.

Reply:
The initial idea was to have this RPC call the setter, but I guess that's not very necessary 🤔

Reply:
@laushinka I like @easyCZ suggestions as they foster loose coupling. 
More context from yesterday's discussion: https://gitpod.slack.com/archives/C02EN94AEPL/p1659600593291589?thread_ts=1659591950.914839&cid=C02EN94AEPL

Reply:
Will update it to one of your suggestions 👍🏼

Reply:
The flip side is that this API is a private API. We can more-less do whatever we want. If you see value in limit scope and adding an rpc that has a very explicit purpose then that's also OK, but we should have a reason for it. It's a question what you're after, and the constraints imposed on you.

Reply:
Done, thanks!